src/frontend/constants/index.ts

```diff
@@ -3,3 +3,11 @@
  */
 
 export const WEBSITE_URL = 'https://certo.schroedinger-hat.org'
+
+export const headerNavLinks = [
```

Comment:
Let's rename this to `HEADER_NAV_LINKS` to maintain the constants naming in sync with the others